Documentation/toc.yml

```diff
@@ -3,8 +3,10 @@
   items:
   - name: Upgrading from HTK
     href: HTKToMRTKPortingGuide.md
-  - name: Updating from RC2 to GA
-    href: UpdatingToGA.md
+  - name: Updating from RC releases
```

Comment:
>RC releases [](start = 24, length = 11)

We're specifically covering upgrading from RC2 to 2.0.0, not any arbitrary version to 2.0.0 - we should be honest about what this guidance is (so people don't try to go from RC1 and expect these instructions will necessarily work)

Reply:
sounds good. i'll add the 2 back :)

Reply:
fixed